### DISCUSSION TITLE
Remove tags from items.

### DISCUSSION THREAD

**Discussion by SenhorPatolino** [FakeFishGames/Barotrauma#9261] [Ideas and suggestions]
**Description**
I dont know if it would cause any crash but if it doesn't i think would be quite nice to remove tags from items, even if that makes the item useless, my example is removing the "smallitem" tag from the revolver and replacing it with a "mediumitem" tag so the player uses two hands instead.

_Just to make clear, its remove the tags on the sub editor, because right now you cant remove any tags that comes by default._

**Comment by krwb** [comment]
I'm totally for this change and would love to see it happen!

**Comment by SquidingTin** [comment]
the tags dont change how many hands it requires to hold it, thats in the xml and not the tags of the xml
`<Holdable slots="Any,RightHand,LeftHand" >` this is for 1 handed
`<Holdable slots="Any,RightHand+LeftHand" >` this is for 2 handed
Hopefully you understand that tags wont effect this and the smallitem tag just means where that item can be placed, such as in cabinets or such

**Comment by SenhorPatolino** [comment]
Wow I even forgot about this, can't answer with confidence now, but at least I think back when I posted this, changing items from "mediumitem" to "smallitem" made possible to hold with 1 hand, and put in lockers that it didn't allow before, this is extremely outdated and I don't have any knownledge about how it works now so its safe to close
